src/tests/ftest/osa/offline_drain.py

```diff
@@ -143,6 +143,7 @@ def run_offline_drain_test(self, num_pool, data=False, oclass=None, pool_fillup=
                     self.run_ior_thread("Read", oclass, test_seq)
                     self.run_mdtest_thread(oclass)
                     self.container = self.pool_cont_dict[self.pool][0]
+                    self.container.daos.env['UCX_LOG_LEVEL'] = 'error'
```

Comment:
Is this just debug for now, or intended to be merged? I could provide a more proper config solution if this needs to be merged

Reply:
@daltonbohning  - Please take a look the ticket to help determine if this is an okay fix. The problem is UCX issues a warning log message to the stdout, and the json parser used by the CI fails the test due to the unexpected output from the test execution. Phil gave me this as one of two solutions to set the UCX_LOG_LEVEL to a level that would not emit the warning. https://daosio.atlassian.net/browse/DAOS-15490

Reply:
Since Phil gave this solution, let's go with that. Ideally, it's better to set envs in the config instead of in python. But I think that needs some extra logic I don't have off the top of my head :)